### DISCUSSION TITLE
Driving Robot Grounded

### DISCUSSION THREAD

**Discussion by MikasaSucasa** [frc5567/2023-Robot#192] [Team Posts]
@JooshO The vision/practice chassis should not be used until the falcons have been inspected and had loctite added to prevent commonly seen issues.

**Comment by MikasaSucasa** [comment]
@coverbeek

**Comment by coverbeek** [comment]
https://www.chiefdelphi.com/t/psa-falcon-failure-mode/378128 ??

**Comment by MikasaSucasa** [comment]
yea
